mx.truffleruby/mx_truffleruby.py

```diff
@@ -294,13 +299,55 @@ def ruby_testdownstream_sulong(args):
     jt('test', 'mri', 'zlib')
     jt('test', 'bundle')
 
+
 def mx_post_parse_cmd_line(opts):
     if mx.suite("tools", fatalIfMissing=False) and not _suite.isBinarySuite():
         mx.project('truffleruby-bin').buildDependencies += [
             mx.distribution('CHROMEINSPECTOR'),
             mx.distribution('TRUFFLE_PROFILER')
         ]
 
+
+mx_sdk.register_graalvm_component(mx_sdk.GraalVmLanguage(
+    name='Truffle.Ruby',
+    short_name='rby',
+    dir_name='ruby',
+    documentation_files=['extracted-dependency:truffleruby:TRUFFLERUBY_GRAALVM_DOCS/README_TRUFFLERUBY.md'],
+    license_files=['link:<support>/GraalCE_Ruby_license_3rd_party_license.txt'],
+    third_party_license_files=[],
+    truffle_jars=[
+        'dependency:truffleruby:TRUFFLERUBY',
+        # 'dependency:truffleruby:TRUFFLERUBY-LAUNCHER', ??
```

Comment:
That's no longer a dependency of TRUFFLERUBY, it's only used by the launcher itself, so it's OK to remove it.